### DISCUSSION TITLE
mini.nvim highlights

### DISCUSSION THREAD

**Discussion by filthyhabits** [nvim-mini/mini.nvim#1334] [Q&A] [answered]
I am using mini.icons, mini.jump2d and mini.move on three machines, intel mac, intel arch and m1 mac, all with the same nvim Lazy based config in identically configged Wezterms.
I noticed on the m1 mac that mini.jump2d is using White on Black highlights for the jump spots while on the other two machines it uses `#b6927b`.
When checking the highlights with `:Telescope highlights` I can see that on the intel machines there is a long list of Mini highlight groups and that for MiniJump2dSpot it says: `MiniJump2dSpot xxx cterm=bold,nocombine gui=bold,nocombine guifg=#b6927b`.
On the m1 `:Telescope highlights` only shows 4 Mini highlight groups, the MiniJump2d highlights, and for MiniJump2dSpot it says:
`MiniJump2dSpot xxx cterm=bold,nocombine gui=bold,nocombine guifg=White guibg=Black`.
Does anybody have an idea how this can happen?
Thanks

**Comment by echasnovski** [comment]
> When checking the highlights with `:Telescope highlights` I can see that on the intel machines there is a long list of Mini highlight groups and that for MiniJump2dSpot it says: `MiniJump2dSpot xxx cterm=bold,nocombine gui=bold,nocombine guifg=#b6927b`.

This is a good indicator that on those machines you use a color scheme that explicitly defines all 'mini.nvim' highlight groups. Judging by the specific color, it is a 'rebelot/kanagawa.nvim' (specifically "dragon" flavor).

> On the m1 `:Telescope highlights` only shows 4 Mini highlight groups, the MiniJump2d highlights, and for MiniJump2dSpot it says:
> `MiniJump2dSpot xxx cterm=bold,nocombine gui=bold,nocombine guifg=White guibg=Black`.

This is an indication that the used color scheme doesn't explicitly define all 'mini.nvim' highlight groups and use the ones which modules themselves define. Usually it is through linking to some existing highlight group, but `MiniJump2dSpot` is a special case with the goal in mind to achieve maximum contrast.

------

If you are *particularly* sure that you use the same color scheme on all three machines, then the Mac one doesn't have the most recent version. The support was [merged on June 24](https://github.com/rebelot/kanagawa.nvim/pull/57).

**Reply by filthyhabits** [reply, marked as answer]
Thank you for your reply, and the great mini modules!
I'm using `sho-87/kanagawa-paper.nvim`.
I emptied my `.local/share/nvim` and after that did a Lazy install. That solved the issue.
